ARCHITECTURE.md

```diff
@@ -41,4 +42,16 @@ For more details about the models we use in Scanner see web-monitoring-db's [API
 - web-monitoring-processing and web-monitoring-versionista-scraper are manually deployed to servers on AWS.
 - We plan to re-deploy these to AWS as a Kubernetes cluster, **but that doesn’t exist yet.**
 
+## Web Page Snapshotting/Capturing Workflow
```

Comment:
This needs to be preceded by `<a id="3"></a>` for the table-of-contents link to work (or you should change the link to `#web-page-snapshottingcapturing-workflow`).

Reply:
Thanks! Will fix